ambry-clustermap/src/main/java/com/github/ambry/clustermap/PropertyStoreToDataNodeConfigAdapter.java

```diff
@@ -105,6 +106,25 @@ public List<String> getAllDataNodeNames() {
     return names == null ? Collections.emptyList() : names;
   }
 
+  /**
+   * Get path to ZNode
+   * @param instanceName the name of instance
+   * @return path to ZNode
+   */
+  private String getPath(String instanceName) {
```

Comment:
i feel private is too strong for this, public or protected would be better

Reply:
Sure. Agree.